tidb-cloud/ticloud-cluster-create.md

```diff
@@ -0,0 +1,51 @@
+---
+title: ticloud cluster create
+Summary: The reference of `ticloud cluster create`.
+---
+
+# ticloud cluster create
+
+Create one cluster in the specified project
+
+```shell
+ticloud cluster create [flags]
+```
+
+## Examples
+
+Create a cluster in interactive mode:
+
+```shell
+ticloud cluster create
+```
+
+Create a cluster in non-interactive mode:
+
+```shell
+ticloud cluster create --project-id <project-id> --cluster-name <cluster-name> --cloud-provider <cloud-provider> --region <region> --root-password <password> --cluster-type <cluster-type>
+```
+
+## Flags
+
+| Flag                    | Description                                                 | Required | Extra                             |
+|-------------------------|-------------------------------------------------------------|----------|-----------------------------------|
+| --cloud-provider string | Cloud provider, one of [AWS]                                | Yes      | Only work in non-interactive mode |
```

Comment:
Does "one of [AWS]" mean that TiDB Cloud CLI only supports creating clusters in AWS,  not in Google Cloud?

Reply:
Currently users could only use CLI to create **serverless clusters**. Serverless clusters only support AWS.